sw/device/tests/rstmgr_sw_rst_ctrl_test.c

```diff
@@ -25,7 +25,7 @@
  *  RSTMGR SW_RST_CTRL Test
  *
  *  This test checks RSTMGR.SW_RST_CTRL_N[index] with peripheral devices.
- *  There are 8 RSTMGR.SW_RST_CTRL_N registers and each register
+ *  There are 7 RSTMGR.SW_RST_CTRL_N registers and each register
```

Comment:
@jdonjdon 
FYI

Reply:
ack,
I will update rstmgr_sw_rst_ctrl_test.c accordingly after this PR is merged